src/main/java/com/google/api/codegen/nodejs/NodeJSGapicContext.java

```diff
@@ -151,7 +151,9 @@ private String returnTypeComment(Method method, MethodConfig config) {
         + callbackType
         + "} callback\n"
         + "  The function which will be called with the result of the API call.\n"
-        + "@returns {gax.EventEmitter} - the event emitter to handle the call\n"
+        + "@returns {"
+        + (config.isBundling() ? "gax.BundleEventEmitter" : "gax.EventEmitter")
+        + "} - the event emitter to handle the call\n"
```

Comment:
Yes, it's still a BundleEventEmitter because it does not change the class, but it's already scheduled, so its method (like runNow()) does nothing with the executor. Added comment to explain this.